1.12/Dockerfile.rhel7

```diff
@@ -0,0 +1,73 @@
+FROM rhscl/s2i-core-rhel7:1
+
+# RHSCL rh-nginx112 image.
+#
+# Volumes:
+#  * /var/opt/rh/rh-nginx112/log/nginx/ - Storage for logs
+
+EXPOSE 8080
+EXPOSE 8443
+
+ENV NGINX_VERSION=1.12
+
+ENV SUMMARY="Platform for running nginx $NGINX_VERSION or building nginx-based application" \
+    DESCRIPTION="Nginx is a web server and a reverse proxy server for HTTP, SMTP, POP3 and IMAP \
+protocols, with a strong focus on high concurrency, performance and low memory usage. The container \
+image provides a containerized packaging of the nginx $NGINX_VERSION daemon. The image can be used \
+as a base image for other applications based on nginx $NGINX_VERSION web server. \
+Nginx server image can be extended using source-to-image tool."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Nginx 1.12" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.expose-services="8443:https" \
+      io.openshift.tags="builder,nginx,rh-nginx112" \
+      com.redhat.component="rh-nginx112-docker" \
+      name="rhscl/nginx-112-rhel7" \
+      release="1"
```

Comment:
Same problem as mentioned above, simple `s/release/version/` should fix the issue.